tests/kernel/arm_runtime_nmi/src/arm_runtime_nmi.c

```diff
@@ -15,6 +15,11 @@
 #include <ztest.h>
 #include <tc_util.h>
 
+/* on v8m arch the nmi pend bit is renamed to pend nmi map it to old name */
+#ifndef SCB_ICSR_NMIPENDSET_Msk
```

Comment:
Minor: should we have #ifdef v8m guards around this macro definition?

Reply:
I think it's ok w/o them.  We will clearly have a build failure if both things aren't defined.

Reply:
LGTM